includes/PSpellEngine.php

```diff
@@ -49,7 +49,11 @@ public function getSuggestions($lang, $words) {
 		$outWords = array();
 		foreach ($words as $word) {
 			if (!pspell_check($plink, trim($word))) {
-				$outWords[] = utf8_encode($word);
+				$outWords[$word] = array();
+
+				foreach( pspell_suggest($plink, trim($word)) as $suggestedWord ) {
+					$outWords[$word][] = utf8_encode($suggestedWord);

```

Comment:
If pspell already outputs utf-8 things will be double encoded, resulting in broken output